extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,47 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors -->
+<!-- SPDX-License-Identifier: MIT -->
 <component type="desktop-application">
   <id>org.alacritty.Alacritty</id>
-  <!-- Translators: The application name -->
+
+  <developer id="org.alacritty">
+    <name>Christian Duerr</name>
+  </developer>
```

Comment:
```suggestion
  <developer_name>Christian Duerr</developer_name>
  <developer id="org.alacritty">
    <name>Christian Duerr</name>
  </developer>
```

Please, also keep the developer_name tag for compatibility with older parsers. Newer ones safely ignore it.

Reply:
Or maybe don't. This is probably up to you - if you want backwards compatibility with more conservative distros such as Linux Mint.